Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -1,37 +1,767 @@
+        - Security
+  items:
+    - LPPWeaponPistolM1911
+
+- type: loadout
+  id: LoadoutSecuritySoleil
+  category: JobsSecurity
+  cost: 1
+  requirements:
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 18000 # 1 hours
```

yamllint

[warning] too few spaces before comment

(comments)